lib/module.js

```diff
@@ -46,7 +46,10 @@ module.exports = function sentry (moduleOptions) {
         '.nuxt/dist/client/img'
       ],
       urlPrefix: publicPath.startsWith('/') ? `~${publicPath}` : publicPath,
-      configFile: '.sentryclirc'
+      configFile: '.sentryclirc',
+      release: {
+        auto: process.env.SENTRY_AUTO_ATTACH_COMMITS || true
```

Comment:
Aren't env vars always strings? Thus `process.env.SENTRY_AUTO_ATTACH_COMMITS`, when set to `false` will have a `"false"` string and will also evaluate to `true`.

Reply:
I stole it from here: https://github.com/nuxt-community/sentry-module/blob/master/lib/module.js#L21

Reply:
It's actually being used like that evereywhere:
https://github.com/nuxt-community/sentry-module/blob/984bae6a41f99395221d7cf4cc0d1f04686ac8c5/lib/module.js#L15-L23

Reply:
Yes, but since those default to `false` (apart from `SENTRY_INITIALIZE` which will probably have the same issue), those will actually work by accident - setting ENV to something will make it evaluate to truthy value thus overriding `false`.

Even if it would not be a string (which it is), you wouldn't be able to override it with `false` as `false || true` still evaluates to `true`.

Reply:
True. Fixed